config.sample.py

```diff
@@ -25,5 +26,15 @@
 SQLALCHEMY_DATABASE_URI = 'sqlite:///{}'.format(os.path.join(os.getcwd(), "data.db"))
 ZOO_DATABASE_URI = 'mysql+pymysql://user:pass@host/database'
 
+# OIDC Config
```

Comment:
Please also add some relevant info to the README in the setup instruction section.